skimage/restoration/_denoise.py

```diff
@@ -394,7 +394,8 @@ def _wavelet_threshold(img, wavelet, threshold=None, sigma=None, mode='soft'):
     return pywt.waverecn(denoised_coeffs, wavelet)
 
 
-def denoise_wavelet(img, sigma=None, wavelet='db1', mode='soft'):
+def denoise_wavelet(img, sigma=None, wavelet='db1', mode='soft',
+                    multichannel=False):
```

Comment:
In the docstring, please, change expected data type for `img` to smth like `(M[, N[, ..., P]][, C])`, and add description for `multichannel` parameter.